README.md

```diff
@@ -3,11 +3,49 @@
 This repo contains sample code for a simple chat webapp that integrates with Azure OpenAI. Some portions of the app use preview APIs and features that should not be used for any production dependency.
```

Comment:
We can delete the comment on the previews API or if we want to leave it link to the Azure preview policy without the claim on production dependency